app/include/zmk/split/bluetooth/service.h

```diff
@@ -20,6 +20,7 @@ struct sensor_event {
 
 struct zmk_split_run_behavior_data {
     uint8_t position;
+    uint8_t source;
```

Comment:
So, putting this in the *middle* of this packed data for sure could cause issues for this change and one or the other sides being out of sync. However, reviewing the code in `service.c`, I see that either way, when we grow this structure, if you flash only the central side, the peripheral side will drop/ignore the run GATT write due to the size being bigger than expected, which is probably a reasonable failure mode for this.

Reply:
I see, thanks for that -- I haven't tested backwards compatibility. Do you still prefer I move this to the end, or keeping as is?

Reply:
As is should be fine.